src/main/java/edu/kis/powp/jobs2d/drivers/adapters/Job2dAdapter.java

```diff
@@ -1,17 +1,32 @@
-package edu.kis.powp.jobs2d.drivers.adapter;
+package edu.kis.powp.jobs2d.drivers.adapters;
 
 import edu.kis.legacy.drawer.panel.DrawPanelController;
 import edu.kis.legacy.drawer.shape.ILine;
 import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.jobs2d.Job2dDriver;
-
+import static edu.kis.powp.jobs2d.features.DrawerFeature.getDrawerController;
 /**
  * driver adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class Job2dAdapter extends DrawPanelController implements Job2dDriver{
```

Comment:
The name is better but it does not indicate the purpose of the adapter or what was actually adapted.
Inheritance is not used (no super drawLine calls). Should be removed!